### ISSUE TITLE
How to find a working powersupply?

### ISSUE THREAD

**Issue by Kilsgaard** [wagiminator/ATtiny814-USB-PD-Adapter#2]
Hi Stefan
Thanks for all your fantastic projects :-)

Have played with this adapter, and think i have it working it works if i connect it to an USB-C on my computer that i used to power the 3v when i programmed it.

But i cannot find any USB-C supply that can get it to power on.

I know that laptop supplies is not made to be correct PD, but i have tried some that should be real PD.
Like this:
![image](https://github.com/wagiminator/ATtiny814-USB-PD-Adapter/assets/11911747/2eecf483-1259-4fa8-a142-2370ef89699e)

Do you have any hint, idea ?

/Graves

**Comment by wagiminator** [comment]
This power adapter should work. There is probably a problem with the board or the firmware. Where exactly is the mistake? Does it output no voltage or the wrong voltage? Is anything displayed on the OLED? Is the microcontroller powered?

**Comment by Kilsgaard** [comment]
It is a bit weirdly. 
Hope I describe this right, I am not that prof at electronics (I am a software guy).

When I connect the board to the USB-C on my compyter, the ATTiny boots.
- I have programmed the ATTiny by powering through my computer.
- It boots up in the software and i can select output power.

When I connect to the PSU:
- The ATTiny does not boot 
- I cannot measure any voltage on the ATTiny.

I have not tried to measure on the output yet.
- I have tried to measure on the inputs and cannot measure any voltages.
  But this can also be my lack of electronic skils, not knowing where to start measuring.

**Comment by wagiminator** [comment]
Do I understand this correctly, if you connect the board to your computer via USB-C then it works, but if you connect it to the power adapter then it doesn't work? Is the CH224K soldered properly (especially the bottom of the chip must be soldered to the corresponding pad on the board)?

**Comment by Kilsgaard** [comment]
Thanks :-)
You were absolutely right, I did not solder the CH224K on the bottom, forgot it.
- Did not have a hotair gun when I assembled the board and forgot all about it.

Took the chip off and soldered it with hotair on the bottom, and everything is perfect now.
- Looks like the board works perfect now

Thanks for the help, and sorry for my stupid mistake (If we meet somewhere, beers is on me :-)
/Graves
